test/use-cases/demographics.integration.js

```diff
@@ -0,0 +1,231 @@
+/* global describe,before,it */
+
+'use strict';
+
+/* eslint max-len: 0 */
+
+process.env.NODE_ENV = 'test';
+
+const _ = require('lodash');
+const chai = require('chai');
+const moment = require('moment');
+
+const Generator = require('../util/generator');
+const SharedIntegration = require('../util/shared-integration');
+const RRSuperTest = require('../util/rr-super-test');
+const exampleSurveys = require('../fixtures/example/survey');
+const config = require('../../config');
+
```

Comment:
Add these lines to see some of the issues in the code
const surveyCommon = require('../util/survey-common');
const SurveyHistory = require('../util/survey-history');